_data/conferences.yml

```diff
@@ -75,7 +75,7 @@
   year: 2019
   id: cvpr19
   link: http://cvpr2019.thecvf.com/
-  deadline: "2017-11-16 23:59:59"
+  deadline: "2019-11-16 23:59:59"
```

Comment:
It should be should be **2018-11-16**, not 2019.

Reply:
Sorry, my bad. Fixed now .